src/brevitas/graph/calibrate.py

```diff
@@ -61,6 +61,26 @@ def __exit__(self, type, value, traceback):
         self.disable_quant_inference.apply(self.model, is_training=self.previous_training_state, quantization_enabled=True)
 
 
+class bias_correction_mode():
```

Comment:
calibration mode inherits from object, this one doesn't. Empty () are also not proper syntax